### ISSUE TITLE
`AboutModal` legal and copyright text color contrast minimum requirements

### ISSUE THREAD

**Issue by SimonFinney** [carbon-design-system/ibm-cloud-cognitive#1035]
#### Affected package

- [x] Cloud & Cognitive

#### Description

IBM Equal Access Accessibility Checker is surfacing a violation for legal and copyright text color contrast - I'm happy to submit the PR to resolve this:

![AboutModal](https://user-images.githubusercontent.com/3846874/126340349-23deb534-ad87-443f-892c-f0a4c681a217.png)
